test/wallet-rpc-test.js

```diff
@@ -318,6 +321,90 @@ describe('Wallet RPC Methods', function() {
     });
   });
 
+  it('should rpc importpubkey rescan = false', async () => {
+    // creating new watchOnly wallet
+    const accountKey = xpub2.xpubkey(network.type);
+    const response = await wclient.createWallet('watchonly', {
+      watchOnly: true,
+      accountKey: accountKey
+    });
+
+    assert.equal(response.id, 'watchonly');
+    assert.equal(response.watchOnly, true);
+
+    const wallet = await wclient.wallet('hot');
+    const keyPair = await wallet.createAddress('default');
```

Comment:
its not really a "key pair" which usually implies a private / public key pair.

Maybe lets just simplifiy!
will this work? (youll have to change `pubkey` to `publicKey` further down....)
```js
const {address, publicKey} = await wallet.createAddress('default');
```

Reply:
yes, this will work